sources/platform/actors/development/quick_start/start_locally.md

```diff
@@ -1,89 +1,131 @@
+Visit [Apify CLI documentation](https://docs.apify.com/cli/) for more information regarding installation and advanced usage.
 
-## 2. Create your Actor
+## Create your Actor
 
-You can use the following command to create an Actor:
+To create a new Actor, use the following command:
 
 ```bash
 apify create
 ```
 
-You will be prompted to name your Actor (e.g, my-actor), and choose the programming language you would like to use (JavaScript, TypeScript, or Python). Next, you will be presented with a list of Actor development templates. After choosing your preferred template the CLI will:
+The CLI will prompt you to:
 
-- Create a `my-actor` directory containing boilerplate code.
-- Install all the project dependencies.
+1. _Name your Actor_: Enter a descriptive name for your actor, such as `my-actor`
+1. _Choose a programming language_: Select the language you want to use for your Actor. (JavaScript, TypeScript, or Python)
+1. _Select a development template_: Choose a template from the list of available options.
+
+After selecting the template, the CLI will:
+
+- Create a `my-actor` directory with the boilerplate code.
+- Install all project dependencies
 
 ![Creation](./images/actor-create.gif)
 
-Finally, switch to your newly created Actor directory:
+Navigate to the newly created Actor directory:
 
 ```bash
 cd my-actor
 ```
 
-## 3. Explore the source code in your editor
+## Explore the source code in your editor
 
-Next, let's explore the Actor's source code. We will use the "PuppeteerCrawler in JavaScript" template code as an example, but all Actor templates follow a similar organizational pattern. The important parts are:
+After creating your Actor, explore the source code in your preferred code editor, We'll use the `PuppeteerCrawler in JavaScript` template code as an example, but all Actor templates follow a similar organizational pattern. The important directories and filer are:
 
-- `src` directory and, namely, the `src/main.js` file, which is the actual code of an Actor.
-- `actor` directory containing Actor's definition files such as `actor.json` and `Dockerfile`. These are important once you deploy your code to the Apify platform.
-- `storage` directory containing the local emulation of [Apify Storage](../../../storage), namely [key-value store](../../../storage/key-value-store), [dataset](../../../storage/dataset), and [request queue](../../../storage/request-queue).
+### `src` Directory
 
-![Actor source code](./images/actor-local-code.png)
+- `srx/main.js`: This file contains the actual code of your Actor
 
-> The following chapters will teach you more about an Actor's structure and files.
+### `actor` Directory
 
-## 4. Run it locally
+- `actor/json`: This file diefines the Actor's configuration, such as input and output specifications.
+- `Dockerfile`: This file contains instructions for building the Docker image for your Actor.
```

Comment:
How about improving the linking here too? (I hard-coded the links there for review purposes)

```suggestion
- `[actor/json](https://docs.apify.com/platform/actors/development/actor-definition/actor-json)`: This file defines the Actor's configuration, such as input and output specifications.
- `[Dockerfile](https://docs.apify.com/platform/actors/development/actor-definition/dockerfile)`: This file contains instructions for building the Docker image for your Actor.
```

BTW, I noticed there is a typo (`diefines` - `defines`). How is it possible that it wasn't picked up by a spell-check extension? 👀

Reply:
good catch, let me investigate around in Vale and my own setup.